lib/tunnel.js

```diff
@@ -96,13 +97,25 @@ function Tunnel() {
               self.binaryPath = binaryPath;
               // Run binary after getting this and attempting atmost 5 times
               console.log(`Starting tunnel`);
-              runBinary_(self, 5, function(e, response) {
-                if (e) {
-                  return reject(e);
-                } else {
-                  return resolve(response);
+              if (options.detachedMode === 'true') {
+                console.log("Running Tunnel in Detached Mode")
+                let tunnelStatus = runBinaryV2_(self, 5)
```

Comment:
Please create a constant for 5 and name it like MAX_TUNNEL_START_RETRY.
magic numbers in code is not good pratice.

Reply:
Done